### PR TITLE
support screen readers

### PR DESCRIPTION
**Pull request by katrinleinweber**
See #175.

### CODE REVIEW

File: _episodes/06-style-guide.md [hunk 1]
```diff
@@ -77,6 +77,13 @@ When making reference to a keyboard key combination that the reader should press
...
 For example, "press <kbd>Ctrl</kbd>+<kbd>X</kbd> to quit nano" should be write as
 
+
+## Links
+
+Please label links with meaningful texts, in order to [improve
+accessibility](https://webaccess.berkeley.edu/ask-pecan/click-here). Please avoid
+`click here` or similar.
+
 ~~~
```
Comment (rgaiacs) [inline _episodes/06-style-guide.md]: This need to be before the new section.

Comment (katrinleinweber) [reply _episodes/06-style-guide.md]: My apologies! I mistook the `~~~` for a footer marker m-(

**Comment by rgaiacs** [comment]
@katrinleinweber Thanks. Looks great.

**Comment by katrinleinweber** [comment]
You're welcome :-)

**Pull request merged**